LenovoLegionToolkit.Lib/Enums.cs

```diff
@@ -53,25 +53,33 @@ public enum FanTableType
 
 public enum FlipToStartState
 {
+    [Display(ResourceType = typeof(Resource), Name = "FlipToStartState_Off")]
```

Comment:
I am thinking that we can probably use the same resource string for all "On" and "Off" cases. No need to repeat the same string multiple times, just have universal "On" and "Off" ones.

Reply:
That would be very clean, yes. My only concern is that some languages might have different endings depending on what On or Off refers to. Then it would be up to the translator to phrase the action in a way that leads to a specific ending.

Reply:
Is there a technical reason to have multiple enums with only On and Off states instead of a single one that is used everywhere?

Reply:
> My only concern is that some languages might have different endings depending on what On or Off refers to.

Yeah, ok fair enough, good point.

> Is there a technical reason to have multiple enums with only On and Off states instead of a single one that is used everywhere?

The design is that all features implement `IFeature<T>` interface and `T` is the type that defines the state of a feature. This was it is easy to decouple UI form specific feature implementation. Check here for example: https://github.com/BartoszCichecki/LenovoLegionToolkit/blob/bca1f818de1a005f287a1a4390ba5a37fe300151/LenovoLegionToolkit.WPF/Controls/AbstractToggleFeatureCardControl.cs#L15 and here: https://github.com/BartoszCichecki/LenovoLegionToolkit/blob/bca1f818de1a005f287a1a4390ba5a37fe300151/LenovoLegionToolkit.WPF/Controls/Dashboard/FnLockControl.cs#L8

Reply:
Aha, I see. Thanks for the clarification!